.github/workflows/release-binaries.yml

```diff
@@ -0,0 +1,188 @@
+name: Deploy binaries and create release
+on:
+  release:
+    types:
+      - published
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: 1
+
+jobs:
+  github_build:
+    name: Build release binaries
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - target: x86_64-unknown-linux-gnu
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-x86_64-unknown-linux-gnu
+
+          - target: x86_64-unknown-linux-musl
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-x86_64-unknown-linux-musl
+
+          - target: i686-unknown-linux-musl
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-i686-unknown-linux-musl
+
+          - target: aarch64-unknown-linux-musl
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-aarch64-unknown-linux-musl
+
+          - target: arm-unknown-linux-musleabi
+            os: ubuntu-latest
+            name: prometheus_wireguard_exporter-arm-unknown-linux-musleabi
+
+          # - target: armv7-unknown-linux-musleabi
+          #   os: ubuntu-latest
+          #   name: prometheus_wireguard_exporter-armv7-unknown-linux-musleabi
+
+          - target: x86_64-apple-darwin
+            os: macOS-latest
+            name: prometheus_wireguard_exporter-x86_64-apple-darwin
+
+          # - target: aarch64-apple-darwin
+          #   os: macOS-latest
+          #   name: prometheus_wireguard_exporter-aarch64-apple-darwin
```

Comment:
Does this one not work? Is there any reason why it's commented?

Reply:
Same here.